jaxrs/src/test/java/org/jboss/as/jaxrs/JaxrsSubsystemTestCase.java

```diff
@@ -1,62 +0,0 @@
-/*
-* JBoss, Home of Professional Open Source.
-* Copyright 2011, Red Hat Middleware LLC, and individual contributors
-* as indicated by the @author tags. See the copyright.txt file in the
-* distribution for a full listing of individual contributors.
-*
-* This is free software; you can redistribute it and/or modify it
-* under the terms of the GNU Lesser General Public License as
-* published by the Free Software Foundation; either version 2.1 of
-* the License, or (at your option) any later version.
-*
-* This software is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-* Lesser General Public License for more details.
-*
-* You should have received a copy of the GNU Lesser General Public
-* License along with this software; if not, write to the Free
-* Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
-* 02110-1301 USA, or see the FSF site: http://www.fsf.org.
-*/
-package org.jboss.as.jaxrs;
-
-import java.io.IOException;
-
-import org.jboss.as.subsystem.test.AbstractSubsystemBaseTest;
-import org.junit.Test;
-
-/**
- * @author <a href="kabir.khan@jboss.com">Kabir Khan</a>
- */
-public class JaxrsSubsystemTestCase extends AbstractSubsystemBaseTest {
```

Comment:
This test should not be deleted. We need to make sure that the old parsers work for backward compatibility. But rename it to include the model version 1.0

Reply:
Ok, I've restored JaxrsSubsystemTestCase, @kabir. Is it legitimate to override testSubsystem() as
```
    @Test
    @Override
    public void testSubsystem() throws Exception {
        standardSubsystemTest(null, false);  // <<== don't compare XML
    }
```
to avoid comparing XML, since otherwise it fails with
```
org.junit.ComparisonFailure: expected:<...:jboss:domain:jaxrs:[1].0"/>
> but was:<...:jboss:domain:jaxrs:[2].0"/>
>
```
I see similar usage in legacy tests in other subsystem.s